src/output/tree.rs

```diff
@@ -70,12 +70,13 @@ impl<'module, W: io::Write> Context<'module, W> {
                 Value::Record(value_fields) => self.write_record(value_fields, Some(format_fields)),
                 _ => panic!("expected record"),
             },
-            Format::Repeat(format) | Format::Repeat1(format) | Format::RepeatCount(_, format) => {
-                match value {
-                    Value::Seq(values) => self.write_seq(values, Some(format)),
-                    _ => panic!("expected sequence"),
-                }
-            }
+            Format::Repeat(format)
+            | Format::Repeat1(format)
+            | Format::RepeatCount(_, format)
+            | Format::RepeatUntil(_, format) => match value {
+                Value::Seq(values) => self.write_seq(values, Some(format)),
+                _ => panic!("expected sequence"),
+            },
```

Comment:
Also need to add a case in `write_format`

Reply:
Oops! Always forget that one, gave me an infinite loop last time :sweat_smile: